evap/results/tools.py

```diff
@@ -142,7 +142,7 @@ def get_single_result_rating_result(evaluation):
 
 
 def get_results_cache_key(evaluation):
-    return "evap.staff.results.tools.get_results-{:d}".format(evaluation.id)
```

Comment:
Any idea why we have the explicit base-10 here?

Reply:
We brainstormed and got two points
* uuids would break, which is bad
* this breaks on evaluation.id == None, which is good, cause otherweise, we could have key collisions